wcfsetup/install/files/lib/system/io/HttpFactory.class.php

```diff
@@ -33,15 +34,16 @@ final class HttpFactory
      */
     public static function getDefaultUserAgent(?string $comment = null): string
     {
+        $domain = ApplicationHandler::getInstance()->getApplicationByID(1)->domainName;
         if ($comment) {
             if (!Regex::compile("^[a-zA-Z0-9_:/\\. -]+$")->match($comment)) {
                 throw new InvalidArgumentException("Invalid comment for user agent given.");
             }
 
-            return \sprintf('WoltLabSuite/%s (%s)', \wcf\getMinorVersion(), $comment);
+            return \sprintf('WoltLabSuite/%s %s (%s)', \wcf\getMinorVersion(), $domain, $comment);
         }
 
-        return \sprintf('WoltLabSuite/%s', \wcf\getMinorVersion());
+        return \sprintf('WoltLabSuite/%s %s', $domain, \wcf\getMinorVersion());
```

Comment:
Even with the correct parameter order this will not result in a valid RFC 7231#5.5.3 User-Agent, thus violating the method contract (neither will the version with the comment).